docker-for-windows/troubleshoot.md

```diff
@@ -160,15 +159,26 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
- Your machine must have the following features for Docker Desktop to function correctly:
+Your machine must have the following features for Docker Desktop to function correctly.
 
-1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview)
-   installed and working
+On **WSL 2** and **Windows Home**:
 
-2. Virtualization enabled in the BIOS
+1. Virtual Machine Platform
+2. [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10){:target="_blank" rel="noopener" class="_"}
+3. Virtualization enabled in the BIOS
+4. Hypervisor enabled at Windows startup
+
+![WSL 2 enabled](images/wsl2-enabled.png){:width="600px"}
+
+On **Windows 10**, you can also use Hyper-V with the following features enabled:
```

Comment:
Maybe move the Hyper-V headline from line 182 to this line.

#### Hyper-V

On Windows 10 Pro or Enterprise,  you can also use Hyper-V with the following features enabled:

Reply:
Updated